src/checkout/context/checkout.tsx

```diff
@@ -87,6 +87,8 @@ export const CheckoutContext = React.createContext<CheckoutContextType>(
   DEFAULT_CONTEXT
 )
 
+const CHECKOUT_PARAM_CODE = '757vMPJiCMdTFkO4'
```

Comment:
ummm ... isn't this the `env` you set earlier?

Reply:
No. This is just a random string Nick generated.

Reply:
k. Do we want to expose it on the UI?

Reply:
As in, is there a way to validate it in case Nick wants to generate a different one that doesn't actively involve making a PR to the UI repo?

Reply:
This code is only useful if you have the experiment ID set already. If you don't then finding this code is completely useless. So, if someone has experiment enabled and still knows the query param and this code then they should be able to upgrade and it is intentional.

Reply:
🤷🏽